Gruntfile.js

```diff
@@ -26,6 +27,103 @@ module.exports = function (grunt) {
             'Licensed GPLv2 */\n',
```

Comment:
Is this still the case? We need to migrate this to Apache 2 to be in concordance with EBI rules.